src/arrayOfPeople.js

```diff
@@ -38,7 +38,16 @@
  * @return {number[]}
  */
 function arrayOfPeople(people) {
-  // write code here
+  const uniquePeople = people.reduce((uniquePeople, person) => {
+    const positionOfPerson = element => uniquePeople.indexOf(element);
```

Comment:
It is good to extract repeated logic into a function but here will be better to save the result of an operation(indexOf)